### DISCUSSION TITLE
Any plan on visual formkit designer

### DISCUSSION THREAD

**Discussion by liguokai** [formkit/formkit#928] [Ideas]
End user may use the deisgner to drag & drop a form (with option json)

**Comment by Ayoskilolo** [comment]
Currently building this on an internal project, would create a personal repo for this and share later on.

**Comment by justin-schroeder** [comment]
We do plan to have a first party form builder experience.

**Reply by JL01v01** [reply]
Looking forward to this!

**Reply by edoversb** [reply]
Me too~!
